rtloader/common/builtins/datadog_agent.c

```diff
@@ -592,11 +587,11 @@ static PyObject *read_persistent_cache(PyObject *self, PyObject *args)
     }
 
     char *v = NULL;
-    Py_BEGIN_ALLOW_THREADS
-    v = cb_read_persistent_cache(key);
+    Py_BEGIN_ALLOW_THREADS v = cb_read_persistent_cache(key);
     Py_END_ALLOW_THREADS
 
-    if (v == NULL) {
+        if (v == NULL)
```

Comment:
I think this should be de-dented right?
```suggestion
    if (v == NULL)
```

Reply:
Indeed, I reverted all the style modification that my editor was too eager to do :sweat_smile: 

I might open a later PR to apply clang-format to those files but this doesn't belong here